app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,213 @@
+
+@DisplayName("TaskService 클래스")
+class TaskServiceTest {
+    private TaskService taskService;
+    private static final String TASK_TITLE = "test";
+    private static final Long NOT_EXIST_ID = 999999999L;
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+    }
+
+    @Nested
+    @DisplayName("getTasks 메소드는")
+    class Describe_getTasks {
+        @Nested
+        @DisplayName("등록된 할 일이 없다면")
+        class Context_no_have_tasks {
+            @Test
+            @DisplayName("비어있는 리스트를 리턴합니다")
+            void it_returns_emptyTasks() {
+                final List<Task> tasks = taskService.getTasks();
+                assertThat(tasks).isEmpty();
+            }
+        }
+
+        @Nested
+        @DisplayName("등록된 할 일이 있다면")
+        class Context_have_tasks {
+            Task task1, task2;
+
+            @BeforeEach
+            void prepareTasks() {
+                task1 = new Task();
+                task1.setTitle("test");
+                task2 = new Task();
+                task2.setTitle("test");
+                taskService.createTask(task1);
+                taskService.createTask(task2);
+            }
+
+            @Test
+            @DisplayName("할 일들을 리턴합니다")
+            void it_returns_tasks() {
+                final List<Task> tasks = taskService.getTasks();
+
+                assertThat(tasks.size()).isEqualTo(2);
+                assertThat(tasks).contains(task1, task2);
+            }
+        }
+    }
+
+    @Nested
+    @DisplayName("getTask 메소드는")
+    class Describe_getTask {
+        @Nested
+        @DisplayName("입력받은 id와 일치하는 등록된 할 일이 없다면")
+        class Context_matchId_NotExist {
+
+            @BeforeEach
+            void prepareEmptyTasks() {
+                List<Task> tasks = taskService.getTasks();
+                tasks.clear();
+            }
+
+            @Test
+            @DisplayName("할 일을 찾을 수 없다는 예외를 던집니다")
+            void it_returns_TaskNotFoundException() {
+                assertThatThrownBy(() -> taskService.getTask(NOT_EXIST_ID))
+                        .isInstanceOf(TaskNotFoundException.class);
+            }
+        }
+
+        @Nested
+        @DisplayName("입력받은 id와 일치하는 등록된 할 일이 있다면")
+        class Context_matchId_exist {
+            Task task;
+            Long id;
+
+            @BeforeEach
+            void prepareTask() {
+                task = new Task();
+                task.setTitle("test");
+                Task createdTask = taskService.createTask(this.task);
+                id = createdTask.getId();
+            }
+
+            @Test
+            @DisplayName("할 일을 리턴합니다")
+            void it_returns_task() {
+                Task foundTask = taskService.getTask(1L);
+
+                assertThat(foundTask).isEqualTo(task);
+            }
+        }
+    }
+
+    @Nested
+    @DisplayName("createTask 메소드는")
+    class Describe_createTask {
+        @Nested
+        @DisplayName("제목이 입력되지 않았다면")
```

Comment:
설명을 읽어보면 `""`도 의도하고 있는 것 같습니다. 다음과 같이 테스트 설명을 구체적으로 고쳐봅시다.
```suggestion
        @DisplayName("제목이 null 이라면")
```